mm2src/mm2_net/src/transport.rs

```diff
@@ -26,6 +27,18 @@ pub enum SlurpError {
     Internal(String),
 }
 
+impl From<SlurpError> for JsonRpcErrorType {
+    fn from(err: SlurpError) -> Self {
+        match err {
+            SlurpError::ErrorDeserializing { .. } | SlurpError::Timeout { .. } | SlurpError::Internal(_) => {
```

Comment:
Don't you think `ErrorDeserializing` and `Timeout` should be matched to another `JsonRpcErrorType`?

Reply:
maybe to `InvalidRequest` ? or what do you think

Reply:
`SlurpError::ErrorDeserializing` should be `JsonRpcErrorType::Parse` , `SlurpError::Timeout` should be `JsonRpcErrorType::Transport`. It's easy to figure this out if you check the `slurp_req` fn code.
https://github.com/KomodoPlatform/atomicDEX-API/blob/16579c69f2458ba6428461997ab49c117a1b08d1/mm2src/mm2_net/src/native_http.rs#L40-L42
https://github.com/KomodoPlatform/atomicDEX-API/blob/16579c69f2458ba6428461997ab49c117a1b08d1/mm2src/mm2_net/src/native_http.rs#L46-L48
https://github.com/KomodoPlatform/atomicDEX-API/blob/16579c69f2458ba6428461997ab49c117a1b08d1/mm2src/mm2_net/src/native_http.rs#L13-L26